bolt_unix.go

```diff
@@ -45,10 +45,10 @@ func funlock(f *os.File) error {
 }
 
 // mmap memory maps a DB's data file.
-func mmap(db *DB, sz int) error {
+func mmap(db *DB, sz int, changesize bool) error {
```

Comment:
Oh, I just found that I forgot about `bolt_windows.go` where is no call to `Sync`.
`Sync` looks to be helper method, and call to `Truncate` is the main.
So, I think `changeSize` is semantically better name.

More: calling to `Truncate` looks to also take non-zero time.